configs/sni.yaml.default

```diff
@@ -7,7 +7,7 @@
 # YAML-based Configuration file
 #  Format :
 #  Actions available:
-#    disable_h2 - removes H2 from the protocol list advertised by ATS; parameter required = None, parameters = true or false
+#    http2 - enable or disable HTTP/2 from the protocol list advertised by ATS; parameters = 'on' or 'off' (boolean)
```

Comment:
No longer boolean? What's the default?

Reply:
YAML booleans takes On/Off, True/ False, Yes/No etc. :) I think On/Off makes most sense here in the docs :).